misc/l10n_feedback.md

```diff
@@ -0,0 +1,38 @@
+# Reviewing translations within your community
+
+Mozilla encourages a three-stage review process:
+1. Linguistic review in Pontoon by community.
+2. Technical review by l10n-drivers.
+3. In-context localization testing by community and users.
+
+This document explains how to perform the first stage of the review process: review the work of another member of your community and provide constructive feedback.
+
+## Attitudes around reviewing translations
+
+Reviewing the translation contributions of another localizer can be controversial. Many attach strong feelings to their contributions, making it very difficult for them to receive feedback and make adjustments to the way they contribute. Reviews are opportunities to ensure high quality contributions while mentoring someone to improve their translation skill set. Reviewers should approach their work with these things in mind:
+1. Everyone is volunteering their time and looking to have a positive impact in their community.
+2. Reviews are an opportunity to give clear, actionable feedback about another localizer’s contributions.
+3. Reviews should be based on objective standards agreed upon by the community and made explicit through [the community’s style guide] (https://mozilla-l10n.github.io/styleguides/) and terminology resources rather than personal preferences. This eliminates the potential for subjective feedback or manipulation through reviews, and thus eliminates potential conflicts within the community.
+4. Reviewers should avoid using vague, potentially inflammatory language in their reviews (e.g., “This translation is terrible, fix it“).
+
+Localizers should approach receiving feedback with these things in mind:
+1. Everyone is volunteering their time and looking to have a positive impact in their community.
+2. Reviews are an opportunity to develop and improve your translation skills. They should be welcomed with an open mind.
+3. Reviews are a natural and required part of the localization process.
+
+## Giving constructive feedback
+
+When reviewing someone else’s translations, feedback should be actionable and based on objective standards. [Style guides] (https://mozilla-l10n.github.io/styleguides/) and terminology resources developed within the community with agreement from its members contain those objective standards. Feedback should include the following elements:
+1. Actionable criticism
+2. Reference to where the translation violates the objective standards
+
+For example, if a translation contains a spelling error, a constructive piece of feedback would look like this:
+“There’s a spelling mistake in *moniter*. The correct spelling can be found in either our official terminology or this section of the community style guide.“
```

Comment:
Not only terminology or community style guide: could also just be in the dictionary :P